src/DotLiquid.Tests/ContextTests.cs

```diff
@@ -233,6 +233,19 @@ public void TestVariables()
             Assert.That(_context["nil"], Is.EqualTo(null));
         }
 
+#if NET6_0_OR_GREATER
+        [Test]
+        public void TestVariables_NET60()
+        {
+            var dateOnly = new DateOnly(2013, 9, 10);
+            _context["dateonly"] = dateOnly;
+            Assert.That(_context["dateOnly"], Is.EqualTo(dateOnly));
+            var timeOnly = new TimeOnly(0, 10, 32);

```

Comment:
same ambiguity - which is hour, minutes, seconds - named parameters would disambiguate.